site/content/en/docs/user/kwokctl-snapshot/yaml-recording.demo

```diff
@@ -0,0 +1,31 @@
+# Let's getting started with kwokctl!
+kwokctl create cluster
+
+# Record it to recording.yaml
+kwokctl snapshot record --path recording.yaml &
+
+# Record some change.
+kwokctl scale node --replicas 2
+kubectl create deployment app --image=app --replicas=3
```

Comment:
The same suggestion in the other review about using a context first applies here. So, the below applies before creating a deployment.

```suggestion
kubectl cluster-info --context kwok-kwok
kubectl create deployment app --image=app --replicas=3
```